<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>411</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>411</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>        添加风险地区        </string>
      </attribute>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>地区</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 12pt &quot;微软雅黑&quot;;
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: 3px solid rgb(0,0,0);
	border-radius:10px
}
QPushButton:hover{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_ct1">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_p1">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_c1">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>        移除风险地区       </string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>地区</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 12pt &quot;微软雅黑&quot;;
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: 3px solid rgb(0,0,0);
	border-radius:10px
}
QPushButton:hover{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_p2">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>70</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_c2">
       <property name="geometry">
        <rect>
         <x>178</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_ct2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 3</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>检测管编号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>检测结果</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>阴性</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>阳性</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="button6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;微软雅黑&quot;;
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: 3px solid rgb(0,0,0);
	border-radius:10px
}
QPushButton:hover{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;微软雅黑&quot;;
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: 3px solid rgb(0,0,0);
	border-radius:10px
}
QPushButton:hover{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
   </property>
   <property name="text">
    <string>健康码处理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;微软雅黑&quot;;
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: 3px solid rgb(0,0,0);
	border-radius:10px
}
QPushButton:hover{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
   </property>
   <property name="text">
    <string>风险地区调整</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;微软雅黑&quot;;
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border: 3px solid rgb(0,0,0);
	border-radius:10px
}
QPushButton:hover{
	
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px
}
</string>
   </property>
   <property name="text">
    <string>核酸结果上传</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
